doc/whats_new/v1.1.rst

```diff
@@ -462,6 +462,15 @@ Changelog
   left corner of the HTML representation to show how the elements are
   clickable. :pr:`21298` by `Thomas Fan`_.
 
+:mod:`sklearn.neural_network`
+.............................
```

Comment:
Nit: May we move this header up such that the modules are in alphabetical order?